BEPs/BEP322.md

```diff
@@ -281,8 +331,11 @@ Response:
 ```json
```

Comment:
I mean example of both normal and error response.

Reply:
added